doc/release_notes_1.1.rst

```diff
@@ -0,0 +1,514 @@
+.. _release_notes_1.1:
+
+ACRN v1.1 (June 2019)
+#####################
+
+We are pleased to announce the release of ACRN version 1.1.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline embedded
+development through an open source platform. Check out the :ref:`introduction` for more information.
+All project ACRN source code is maintained in the https://github.com/projectacrn/acrn-hypervisor
+repository and includes folders for the ACRN hypervisor, the ACRN device
+model, tools, and documentation. You can either download this source code as
+a zip or tar.gz file (see the `ACRN v1.1 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.1>`_)
+or use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v1.1
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v1.1 documents can be found at https://projectacrn.github.io/1.1/.
+Documentation for the latest (master) branch is found at https://projectacrn.github.io/latest/.
+ACRN v1.1 requires Clear Linux* OS version 29970 or newer. Please follow the
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 1.1 major features
+**************************
+
+Hybrid Mode Introduced
+======================
+In hybrid mode, a Zephyr OS is launched by the hypervisor even before the Service OS is
+launched (pre-launched), with dedicated resources to achieve highest level of isolation.
+This is designed to meet the needs of a FuSa certifiable safety OS.
+
+More types of Operating systems supported
```

Comment:
@dbkinder  How about rephrasing this to something like: "Support for new guest Operating Systems" ?